pkg/services/notifications/mailer.go

```diff
@@ -25,11 +26,16 @@ func (ns *NotificationService) send(msg *Message) (int, error) {
 	}
 
 	var num int
-	for _, address := range msg.To {
+
+	h := make(map[string][]string)
```

Comment:
This file has changed in master so you have to merge in latest master. Then I suggest:
- putting each message in an array
- extract creating a message to a method
- populate array with either one message or multiple message
- use `dialer.DialAndSend(messages...)` so that only one connection being used to send the emails.